pkg/networks/usernet/client.go

```diff
@@ -127,6 +129,27 @@ func (c *Client) Leases(ctx context.Context) (map[string]string, error) {
 	return leases, nil
 }
 
+// WaitOpeningSSHPort Wait until the guest ssh server is available.
+func (c *Client) WaitOpeningSSHPort(ctx context.Context, inst *limatype.Instance) error {
+	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
It seems that 10 seconds is too short for testing on CI.
Should it make timeout configurable by `LIMA_SSH_OVER_VSOCK_TIMEOUT` for CI?

> time="2025-09-19T03:00:51Z" level=info msg="[hostagent] [VZ] - vm state change: running"
> time="2025-09-19T03:01:01Z" level=warning msg="[hostagent] Failed to wait for the guest SSH server to become available, falling back to usernet forwarder" error="Get \"http://lima/extension/wait_port?ip=192.168.5.15&port=22\": context deadline exceeded"
> ...
> time="2025-09-19T03:01:49Z" level=info msg="[hostagent] Waiting for the essential requirement 1 of 2: \"ssh\""
> time="2025-09-19T03:02:02Z" level=info msg="[hostagent] The essential requirement 1 of 2 is satisfied"